modules/swagger-core/src/main/java/com/wordnik/swagger/jackson/AbstractModelConverter.java

```diff
@@ -35,7 +37,10 @@ protected AbstractModelConverter(ObjectMapper mapper) {
             new SimpleModule("swagger", Version.unknownVersion()) {
               @Override
               public void setupModule(SetupContext context) {
-                context.insertAnnotationIntrospector(new SwaggerAnnotationIntrospector());
+                context.insertAnnotationIntrospector(
+                        new AnnotationIntrospectorPair(
+                                new SwaggerAnnotationIntrospector(),
+                                new JacksonAnnotationIntrospector()));
```

Comment:
I agree, I believe that the JacksonAnnotatinIntrospector is embedded in SwaggerAnnotationIntrospector